docs/howto/openstack/nova/new-server.md

```diff
@@ -0,0 +1,339 @@
+	server is shut off you are still getting charged for it, but less so.) 
+	At any time, this estimated cost is displayed in the green rectangular area 
+	at the top. Something else that affects the cost is the size of the root 
+	device. Take a look at the _Size_ parameter below, and notice the 
+	default (in 
+	[gibibytes](https://en.wikipedia.org/wiki/Gigabyte#Base_2_(binary))). 
+	You may leave the root device size unchanged, or modify it to be lower 
+	or higher than the default.
+		
+	When, at a later time, you decide to delete the server, you can 
+	do so but **keep** its boot volume (you may want, for example, to 
+	attach the exact same volume to a new server). Just leave the _Delete 
+	on termination_ option disabled if you want this kind of flexibility. 
+	On the other hand, if you want your root volume to be automatically
+	deleted when the server terminates, enable _Delete on termination_.
+	Use this option with caution.
+		
+	Finally, you may enable _Disaster recovery_ for the server. If 
+	you do, then daily server snapshots will be created, and you will have 
+	the option for easy and fast rollups to previous snapshots. Please be 
+	aware that enabling this option increases the server's monthly estimated
+	cost (again, it is displayed in the green rectangular area at the top).
+		
+	![Server configuration](assets/new-server/shot-04.png)
+		
+	Regarding networking, select one of the available networks to 
+	attach the new server to. If you want the server accessible from the 
+	Internet, do not forget to enable the _I want an external IP for my 
+	server_ option. In the section below, set _Security Groups_ to 
+	_default_.
+		
+	![Networking and security groups](assets/new-server/shot-05.png)
+		
+	If you already have one or more public keys in your 
+	{{extra.brand}} Cloud account, you can now select a key to be included 
+	in the `~/.ssh/authorized_keys` file of the server's default user. That 
+	way, you can securely log into the remote user's account without typing 
+	a password. If there are no public keys to choose from, activate the 
+	_Password login enabled_ option and set a password for a specific user 
+	(with a username you define).
+		
+	![Login and keypairs](assets/new-server/shot-06.png)
+		
+	A configuration script is automatically prepared based on the 
+	choices you have already made. That script runs during system boot and 
+	performs housekeeping tasks like user account creation, enabling 
+	acceptable authentication methods, and configuring remote package 
+	repositories. Click on _Advanced Options_ to see the default script.
+		
+	![User-data propagation method](assets/new-server/shot-07.png)
+		
+	It is now time to create your {{extra.brand}} Cloud server; 
+	click the green _Create_ button, and the new server will be readily 
+	available in a few seconds.
+		
+	![Create new server](assets/new-server/shot-08.png)
+	
+	## Viewing information about the newly created server	
```

Comment:
Uh, no. Don't create the additional heading *within* the content tab. End the content tabs section here, insert the heading, and then create a new content tabs section that documents the corresponding functionality for the GUI and the CLI.

Reply:
OK, done!